### ISSUE TITLE
[Research] Update Veteran and Staff Pilot Feedback Research Plans/Conversation Guide for Onsite Visit

### ISSUE THREAD

**Issue by kristenmcconnell** [department-of-veterans-affairs/va.gov-team#37949]
# Description
Since remote pilot feedback methods haven't been fruitful in returning results, we are planning to make an on-site visit to St. Louis. Therefore, we need to optimize our research plans and conversation guides for this new research method. 

Current Veteran plan: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/veteran-facing/Martinsburg-Staff-Moderated

Current staff plan: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/staff-facing/st-louis/pilot-feedback

Going on-site can allow us to gain feedback on future features as well. The topics to be added to this research plan are TBD.

# Task
- [x] Update Veteran research plan
- [x] Update Staff research plan
- [x] Identify if any assets need to be created for the future feature sections.

# Acceptance Criteria
- [x] Review plan with UX and product
- [x] Send to platform research ops for review
- [x] Add to research repo (we might need to just update our old repo cards...TBD)

**Comment by kristenmcconnell** [comment]
Sent out research plan for review via Slack: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/35543

**Comment by kristenmcconnell** [comment]
[Draft conversation guide for Veteran intercept](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/veteran-facing/StLouis-pilot-feedback/conversation-guide.md)

**Comment by kristenmcconnell** [comment]
I added a note to [the old remote study](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/veteran-facing/Martinsburg-Staff-Moderated) explaining that it was replaced with this study. I thought that it might be good to leave that research plan on GitHub, so we'll know in the future that we tried it and it wasn't successful.

**Comment by kristenmcconnell** [comment]
[Draft conversation guide for staff interviews while on-site.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/research/staff-facing/st-louis/pilot-feedback/conversation-guide.md)

**Comment by kristenmcconnell** [comment]
Overall research folder for both studies were created on GitHub:
- Staff: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/staff-facing/st-louis/pilot-feedback
- Veteran: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/checkin/research/veteran-facing/StLouis-pilot-feedback

**Comment by kristenmcconnell** [comment]
Research Repo tickets updated: 

- https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/68
- https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues/79

**Comment by kristenmcconnell** [comment]
Now, I need to work with Peter Russo to see if we can add a small number of appointment notification questions to the staff guide.

**Comment by kristenmcconnell** [comment]
Peter is working through some questions, so they can be added within another ticket.

**Comment by kristenmcconnell** [comment]
@loripusey We will need another ticket to add the appointment notification questions to the staff conversation guide once VAOS is complete.

**Comment by loripusey** [comment]
here's a placeholder [story](https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/38840)
